doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
```

Comment:
IMO, the general purpose document would be the 'development story' part. @juimonen has started the work already. I'm the owner of YOCTO 11495

We can keep this doc until other pieces are ready and then see if it makes sense to merge the docs or go with the linking approach.